### PR TITLE
Bug 1989720: Add profile customizations to README

### PR DESCRIPTION
**Pull request by damemi**
Follow up https://github.com/openshift/cluster-kube-descheduler-operator/pull/201 to update readme describing profileCustomizations (and example in sample CR)

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @ingvagabund 
could you please tag? Forgot to add these to the readme

File: README.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ spec:
   deschedulingIntervalSeconds: 1800
   profiles:
   - AffinityAndTaints
+  - LifecycleAndUtilization
+  profileCustomizations:
+    podLifetime: 5m
```
Comment (kasturinarra) [inline README.md:66]: @damemi above does not work, it throws us an error which reads as below, so the value should be only integer

spec.profileCustomizations.podLifetime: Invalid value: "string": spec.profileCustomizations.podLifetime in body must be of type integer: "string"

Comment (damemi) [reply README.md:66]: Opened a new PR to address this: https://github.com/openshift/cluster-kube-descheduler-operator/pull/211

**Pull request merged**